.vscode/cspell.json

```diff
@@ -24,8 +24,11 @@
     "otelcol",
     "otlp",
     "OTLP",
+    "protobuf",
+    "parentbased",
```

Comment:
The string "parentbased" is not a proper spelling, but I suspect that you've added it to the list because it appears as an identifier. Is that correct?

If so, I suggest adding this word locally to the page(s) that contain the identifier. (Search for cSpell and you'll see how I've done this for other pages.

Reply:
yes, I thought I can add it because we hav relref and tabpane there as well, although "parentbased" and "parent based" both might appear, so let me remove it.